### DISCUSSION TITLE
sp_helptext giving a weird indentation problem

### DISCUSSION THREAD

**Discussion by sqlmonk** [microsoft/azuredatastudio#21989] [Q&A]
Today I updated ADS to the newer version and started facing a weird problem. It is not getting resolved even I go back to the old version. 

The sp_helptext is giving me output with the enter charecters and with weird indentation. Below is the screenshot for the same.
<img width="1417" alt="Screenshot 2023-02-21 at 8 06 31 PM" src="https://user-images.githubusercontent.com/10770801/220374812-025021f0-a378-4328-ae00-58a4d19ca6dd.png">

When i copy the output and paste it in another window, the code is not usable at all. Below is the screenshot how it looks like.
<img width="1402" alt="Screenshot 2023-02-21 at 8 07 44 PM" src="https://user-images.githubusercontent.com/10770801/220374941-fd585d1f-43b8-43e4-8766-a3438d31b372.png">

The version of Azure Data Studio is as below on Mac M1
Version: 1.41.2
Commit: 0f0960d0e774e1ce903b79378addff96117c27a6
Date: 2023-02-08T20:18:32.853Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.2.0

Please help as I am stuck badly.

**Comment by sqlmonk** [comment]
I figured that if create a stored procedure by doing a selection of the code and executing it create a problem in sp_helptext.

But if the file contains only the create procedure code and I execute whole file without any selection then the sp_helptext returns the outpu properly.
Is the behaviour intended? Is the SQL server stores the code differently on doing a selection and executing it ?

**Comment by erinstellato-ms** [comment]
Closing this as an issue was opened (https://github.com/microsoft/azuredatastudio/issues/21990).  Please provide additional details on the issue to help us recreate the behavior you're seeing.  Thanks!
